packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetailsMultiple.vue

```diff
@@ -1,21 +1,159 @@
-      <p v-translate>Multiple spaces selected</p>
+  <div id="oc-spaces-details-multiple-sidebar">
+    <div class="spaces-preview oc-mb">
+      <div class="spaces-preview-body">
+        <oc-icon class="preview-icon" size="xxlarge" variation="passive" name="layout-grid" />
+        <p class="preview-text" v-text="selectedSpacesString" />
+      </div>
+    </div>
+    <div>
+      <table class="details-table" :aria-label="detailsTableLabel">
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Total quota:')" />
+          <td v-text="totalSelectedSpaceQuotaTotal" />
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Remaining quota:')" />
+          <td v-text="totalSelectedSpaceQuotaRemaining" />
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Used quota:')" />
+          <td v-text="totalSelectedSpaceQuotaUsed" />
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Enabled:')" />
+          <td v-text="totalEnabledSpaces" />
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Disabled:')" />
+          <td v-text="totalDisabledSpaces" />
+        </tr>
+      </table>
     </div>
   </div>
 </template>
 <script lang="ts">
-import { defineComponent } from 'vue'
+import { formatFileSize } from 'web-pkg/src/helpers'
+import { computed, defineComponent, PropType } from 'vue'
+import { SpaceResource } from 'web-client/src'
+import { useTranslations } from 'web-pkg/src'
 
 export default defineComponent({
-  name: 'SpaceDetailsMultiple'
+  name: 'SpaceDetailsMultiple',
+  props: {
+    selectedSpaces: {
+      type: Array as PropType<Array<SpaceResource>>,
+      required: true
+    }
+  },
+  setup(props) {
+    const { $gettext, $ngettext, $gettextInterpolate, $language } = useTranslations()
+    const totalSelectedSpaceQuotaTotal = computed(() => {
+      let total = 0
+      props.selectedSpaces.forEach((space) => {
+        total += space.spaceQuota.total
+      })
+      return formatFileSize(total, $language.current)
+    })
+    const totalSelectedSpaceQuotaRemaining = computed(() => {
+      let remaining = 0
+      props.selectedSpaces.forEach((space) => {
+        if (space.disabled) {
+          return
+        }
+        remaining += space.spaceQuota.remaining
+      })
+      return formatFileSize(remaining, $language.current)
+    })
+    const totalSelectedSpaceQuotaUsed = computed(() => {
+      let used = 0
+      props.selectedSpaces.forEach((space) => {
+        if (space.disabled) {
+          return
+        }
+        used += space.spaceQuota.used
+      })
+      return formatFileSize(used, $language.current)
+    })
+    const totalEnabledSpaces = computed(() => {
+      let enabled = 0
+      props.selectedSpaces.forEach((space) => {
+        if (!space.disabled) {
+          enabled++
+        }
+      })
+      return enabled
```

Comment:
I'd prefer this for more readability:

```suggestion
      return props.selectedSpaces.filter((s) => !s.disabled).length
```